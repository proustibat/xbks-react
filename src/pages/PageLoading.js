import React from "react";

const PageLoading = () => (
  <div className="loader">
      <img
          className="loader__image"
          src="/images/loader.gif"
          alt="loader animation"
      />
  </div>
);

export default PageLoading;
