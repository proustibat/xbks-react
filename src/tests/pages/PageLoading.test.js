import React from 'react';
import { shallow } from 'enzyme';
import PageLoading from '../../pages/PageLoading';

test( 'Should render loading page correctly', () => {
    const wrapper = shallow( <PageLoading /> );
    expect( wrapper ).toMatchSnapshot();
} );
