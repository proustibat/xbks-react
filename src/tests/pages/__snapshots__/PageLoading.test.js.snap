// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render loading page correctly 1`] = `
<div
  className="loader"
>
  <img
    alt="loader animation"
    className="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
